controllers/controlplane/pki.go

```diff
@@ -0,0 +1,208 @@
+package controlplane
```

Comment:
As discussed this will be moved into the NCP controller and leveraging the kubeadm control plane funcs documented in #29

Reply:
I will remove this file.